dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/process/ProcessServiceImpl.java

```diff
@@ -2965,7 +2967,7 @@ public boolean robTaskGroupResouce(TaskGroupQueue taskGroupQueue) {
 
     @Override
     public boolean acquireTaskGroupAgain(TaskGroupQueue taskGroupQueue) {
-        return robTaskGroupResouce(taskGroupQueue);
+        return robTaskGroupResource(taskGroupQueue);
```

Comment:
I think the method `acquireTaskGroupAgain` can be removed and use `robTaskGroupResouce` directly.

Reply:
Done